.changeset/five-turkeys-double.md

```diff
@@ -0,0 +1,5 @@
+---
+'@cloudfour/patterns': major
+---
+
+Renamed the Heading property from `permalink_id` to `id` and allowed you to use `id` when `permalink` is false.
```

Comment:
This is _technically_ a breaking change, though as far as I can tell we're not using`permalink_id` anywhere yet.